pkg/reposync/conditions.go

```diff
@@ -110,9 +111,8 @@ func SetSyncing(rs *v1beta1.RepoSync, status bool, reason, message, commit strin
 }
 
 // setCondition adds or updates the specified condition with a True status.
-// It returns a boolean indicating if the condition status is transited.
-func setCondition(rs *v1beta1.RepoSync, condType v1beta1.RepoSyncConditionType, status metav1.ConditionStatus, reason, message, commit string, errorSources []v1beta1.ErrorSource, errorSummary *v1beta1.ErrorSummary, lastUpdate metav1.Time) bool {
-	conditionTransited := false
+// It returns a boolean indicating if the condition status has transitioned.
```

Comment:
Update docstring to reflect new signature (two booleans). Ditto for `rootsync`

Reply:
done